src/openrct2/localisation/UTF8.cpp

```diff
@@ -92,6 +92,21 @@ int32_t UTF8Length(const utf8* text)
     return count;
 }
 
+/**
+ * Gets the number of bytes required to store the first x characters in a UTF-8 string
```

Comment:
These aren't characters they are codepoints.

Reply:
Changed to match the previous function